src/arrays/multidimensional_arrays/hard/q9/MainTest.java

```diff
@@ -0,0 +1,74 @@
+package arrays.multidimensional_arrays.hard.q9;
+
+import global.BaseTest;
+import global.MethodTest;
+import global.tools.TestOption;
+import global.variables.Clause;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.StringLiteral;
+import global.variables.wrappers.Optional;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static global.utils.ArrayUtil.generateRandomArray;
+
+public class MainTest extends BaseTest {
+    // Parsons
+
+    public static final int[][][] nums = {{{1, 2}, {3, 4}, {5, 6}}, {{2, 1}, {4, 4}, {11, 0}}, {{12, 24}, {31, 44}, {51, 60}}};
+
+    public Clause[] testSentence() {
+        String st = "";
+        for (int i = 0; i < nums.length; i++) {
+            for (int j = 0; j < nums[i].length; j++) {
+                for (int k = 0; k < nums[i][j].length; k++)
+                    st += nums[i][j][k] + " ";
+            }
+        }
+        return new Clause[]{
+                new StringLiteral(st.trim()),
```

Comment:
get rid of the trim and optional space since this is a parsons and it will always have the extra space at the end